graylog2-web-interface/src/pages/PluggableEventProceduresPage.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+
+import usePluginEntities from 'hooks/usePluginEntities';
+import EventsPageNavigation from 'components/events/EventsPageNavigation';
+
+const PluggableEventProceduresPage = () => {
+  const pluggableEventProcedures = usePluginEntities('eventProcedures');
+
+  const EventProceduresPage = pluggableEventProcedures[0]?.EventProcedures;
+
+  return (
+    <EventProceduresPage navigationComponent={<EventsPageNavigation />} useCoreRoutes />
```

Comment:
We need to make sure that `EventProceduresPage` is not undefined and is of type Component to make sure in any edge case it won't break and throw errors